src/components/Trackpad/FloatingMirror.tsx

```
@@ -0,0 +1,253 @@
+                </button>
+            </div>
+
+            {/* Resize handle (bottom-right) */}
+            <div
+                className="absolute bottom-0 right-0 w-6 h-6 cursor-se-resize touch-none flex items-end justify-end pr-1 pb-1"
+                onTouchStart={onResizeStart}
+                onTouchMove={onResizeMove}
+                onTouchEnd={onResizeEnd}
+            >
+                {/* visual grip dots */}
+                <svg width="10" height="10" viewBox="0 0 10 10" className="opacity-50">
```

Biome (2.4.4)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)